### PR TITLE
V2 : enable multi-gesture touch events for mobile devices

### PR DESCRIPTION
**Pull request by z3dev**
These changes enable multi-gesture touch events for mobile devices.

Sadly, a new package was added, most-gestures, as a bug fix to this package was required. This should be a temporary fix until this package has been updated.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Does your submission pass tests?

Testing ws performed on several PCs, iPad, Android mobile.

### CODE REVIEW

File: packages/most-gestures/src/drags.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+const { merge } = require('most')
+
+// based on http://jsfiddle.net/mattpodwysocki/pfCqq/
+function mouseDrags (mouseDowns$, mouseUps, mouseMoves, settings) {
+  const {pixelRatio} = settings
+  return mouseDowns$.flatMap(function (md) {
```
Comment (SimonClark) [inline packages/most-gestures/src/drags.js:6]: What's the significance of the "$" at the end of the variable name?

**Comment by SimonClark** [comment]
Can we get some sort of indication how this version differs from the original?

File: packages/most-gestures/src/drags.js (hunk 1 above)
Comment (z3dev) [reply packages/most-gestures/src/drags.js:6]: Ok. I’ll add some notes about the fixes

Comment (z3dev) [reply packages/most-gestures/src/drags.js:6]: No idea about the choice of variable names. I suggest we leave all the code as taken, except the changes, and hope there’s a merge to the original repository later.

Comment (SimonClark) [reply packages/most-gestures/src/drags.js:6]: hah, no problem. I thought it might be some convention that I'd never run into before.

**Review by SimonClark — APPROVED** [review]

**Pull request merged**